frontend/src/app/home/home.component.ts

```diff
@@ -5,17 +5,14 @@ import { MidwareService } from '../midware/midware.service';
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
-  styleUrls: ['./home.component.scss']
+  styleUrls: ['./home.component.scss'],
 })
 export class HomeComponent implements OnInit {
+  midObj = {};
```

Comment:
this is not standard Typescript, it should stay as `midObj: Object;`

The = null part is not necessary tho, so that should be removed probably. Overall doesn't really matter, though.